src/Symfony/Component/Console/Tests/Tester/Constraint/CommandIsSuccessfulTest.php

```diff
@@ -26,11 +26,27 @@ public function testConstraint()
         $this->assertTrue($constraint->evaluate(Command::SUCCESS, '', true));
         $this->assertFalse($constraint->evaluate(Command::FAILURE, '', true));
         $this->assertFalse($constraint->evaluate(Command::INVALID, '', true));
+    }
+
+    public function providesUnsuccessful(): iterable
```

Comment:
please put the dataprovider after the test case

Reply:
Done